### PR TITLE
[Nd-33] api games

### PR DESCRIPTION
**Pull request by Reactor94**
Created **Genres** schema.
Created **Game** schema.
Downloaded **Games** and added them to database.
Downloaded **Genres** and added them to database
Downloaded **Covers** and added them to database

### CODE REVIEW

File: backend/src/getCovers.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        method: 'POST',
+        headers: {
+          Accept: 'application/json',
+          'user-key': 'c5070d3b870504a67370014ce197555e',
+        },
+        data: `fields url; where game = (${gamesWithoutCovers}); limit 150;`,
+      })
+        .then(response => {
+          response.data.map(async cover => {
+            await Game.updateMany({ cover: cover.id }, { cover: cover.url });
```
Comment (wtart-softserve) [inline backend/src/getCovers.js]: Try to update all covers at once.

File: backend/src/getGames.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+const axios = require('axios');
+const db = require('./database/database');
+const Game = require('./models/Game');
+
+const getGames = () => {
+  axios({
```
Comment (wtart-softserve) [inline backend/src/getGames.js]: You are using this in a few places, and you could refactor it to something like 
function apiClient(url, data, method = 'POST') {
  return axiosurl: `https://api-v3.igdb.com/${url}`,
    method,
    headers: {
      Accept: 'application/json',
      'user-key': 'c5070d3b870504a67370014ce197555e',
    },
    data
  })
}

File: backend/src/getCovers.js (hunk 1 above)
Comment (Reactor94) [reply backend/src/getCovers.js]: I had an issue when i was trying to download a covers for games and for some reason about 10 games did not get a covers.  And  the code above should download  all covers at once, even if we dont have any cover or we need some covers.

File: backend/src/getCovers.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+const axios = require('axios');
+const fs = require('fs');
+const db = require('./database/database');
+const Game = require('./models/Game');
+
+const getCover = () => {
```
[the remark this answers is not in the document]
Comment (Reactor94) [reply backend/src/getCovers.js]: Renamed to seedGames, seedGenres. getCovers now gets only covers.

File: backend/src/getCovers.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  db.connection.on('open', () => {
+    Game.find({}, function(error, games) {
```
[the remark this answers is not in the document]
Comment (Reactor94) [reply backend/src/getCovers.js]: Now this function only gets covers.

**Pull request merged**